### DISCUSSION TITLE
Bug when trying to stop the prompt.

### DISCUSSION THREAD

**Discussion by oliverbob** [bionic-gpt/bionic-gpt#236] [Q&A]
I tried to install it on an HPE ML380 server (without a GPU). It works on the first few prompts, but when prompts are becoming complex, obviously its taking delay receiving the answer from the default model. Clicking on "Stop Processing" doesn't do anything.

I'm using the official docker and the curl -O https://raw.githubusercontent.com/purton-tech/bionicgpt/main/docker-compose.yml directive.

Is this the default behavior?

Thanks.

**Comment by tbendien** [comment]
You may be overloading the token space. You are probably using a model that is limited to 2048 tokens. Go to Prompt Settings and set history to 0.
